aws/resource_aws_lambda_permission.go

```diff
@@ -394,11 +403,7 @@ func getLambdaPolicyStatement(out *lambda.GetPolicyOutput, statemendId string) (
 		return nil, fmt.Errorf("Error unmarshalling Lambda policy: %s", err)
 	}
 
-	statement, psErr := findLambdaPolicyStatementById(&policy, statemendId)
-	if psErr != nil {
-		return nil, fmt.Errorf("Error finding Lambda policy statement: %s", psErr)
-	}
-	return statement, nil
+	return findLambdaPolicyStatementById(&policy, statemendId)
```

Comment:
re-casting the error to a human readable string is unnecessary since we are a tool that can do something about it.

Reply:
Agreed, there's no need to wrap this again